crates/solidity/outputs/npm/package/package.json

```diff
@@ -55,7 +55,8 @@
         "x86_64-pc-windows-msvc",
         "x86_64-unknown-linux-gnu",
         "x86_64-unknown-linux-musl"
-      ]
+      ],
+      "glibc": "2.28"
```

Comment:
I suggest keeping the `napi` object pure to NAPI's own properties, and not add on our custom config here, and I was confused a bit to how it was read/used by `napi-rs/cli`. Maybe we can put it in a custom top-level property instead?

Reply:
I was torn and was thinking between this or a configuration in a Cargo.toml for the addon but decided grouping it together would be the cleanest. I'll extract it out, then.

Reply:
Extracted in https://github.com/NomicFoundation/slang/pull/909/commits/6414218b63c10f473a9295841397abd35f2b18be to `"slangMetadata": { "targetGlibc": "2.28" }` to be clear that it's our metadata.